test/cpp/test_status_dont_show_cpp_error_context.cpp

```diff
@@ -1,5 +1,8 @@
 #include "test/cpp/test_status_common.h"
 
+using torch_xla::CppErrorContextMode;
```

Comment:
Style: please make the macro body fully qualify the references of these names so that we don't have to do this here.

Reply:
Ah, yes. Sorry about that. Since I couldn't do it with `StatusTest` (i.e. `::torch_xla::StatusTest`), I incorrectly assumed I couldn't do it for `CppErrorContextMode`. It should be fixed, now.